### DISCUSSION TITLE
`null is not an object (evaluating 'document.head.appendChild')`

### DISCUSSION THREAD

**Discussion by jorisre** [remix-run/remix#4796] [Q&A]
Hi!

I get the following error several times per day (thanks to Sentry).

`null is not an object (evaluating 'document.head.appendChild')`

I'm currently not able to reproduce but it seems the error occurs here: https://github.com/remix-run/remix/blob/5783dc00fa27c6198c332096d70ae638f2767e4a/packages/remix-react/links.ts#L295

If someone have an idea why it throws an error?

**Comment by JeffBeltran** [comment]
This has been annoying me like no other, it appears to be related to the handling of the `export function links()` but im not sure
